dll/directx/wine/ddraw/ddraw.c

```diff
@@ -1132,6 +1132,7 @@ static HRESULT WINAPI ddraw7_SetDisplayMode(IDirectDraw7 *iface, DWORD width, DW
 
     /* TODO: The possible return values from msdn suggest that the screen mode
      * can't be changed if a surface is locked or some drawing is in progress. */
+
```

Comment:
This change should be reverted before merging, but please wait with that until the rest of your changes have been reviewed by multiple devs. You intended to highlight the comment above I guess. Good idea.

Reply:
![LetsHaveALook](https://user-images.githubusercontent.com/33393466/192089162-64dbefcb-f102-4721-a2e0-b1630709d27d.PNG)